### DISCUSSION TITLE
'beta_channel' - 'saturation_beta': Ambiguity When Setting Custom Priors for Channels in MMM Configuration

### DISCUSSION THREAD

**Discussion by Denzel124** [pymc-labs/pymc-marketing#1245] [Q&A] [answered]
Good morning!
I’m reaching out to ask for clarification on the PyMC-Marketing documentation regarding the configuration of custom priors in the MMM model. . I’m currently using the library for a marketing mix modeling project and need to set a prior for the channels in the model.
While reviewing the documentation, I found two possible ways to specify the prior in the my_model_config dictionary:
-As shown in the example notebook, using 'saturation_beta'  to refer to the channel prior (https://www.pymc-marketing.io/en/stable/notebooks/mmm/mmm_example.html):
```python
my_model_config = {  
    "saturation_beta": {  
        "dist": "HalfNormal",  
        "kwargs": {"sigma": prior_sigma},  
    },  
}  
```
-Referring to the prior as 'beta_channel' (https://www.pymc-marketing.io/en/stable/api/generated/pymc_marketing.mmm.mmm.MMM.html)
        my_model_config = { "beta_channel": Prior("LogNormal", mu=np.array([2, 1]), sigma=1), }
Could someone confirm which approach is correct for setting custom priors for the channels? Using 'beta_channel' does not seem to affect the posterior predictions, while changing the 'saturation_beta' prior does produce effects on the posterior predictions. My concern lies in the ambiguity of using 'saturation_beta' as the name for the prior associated with the channels.

**Comment by luiztauffer** [comment]
it looks like the notebook is outdated: https://github.com/pymc-labs/pymc-marketing/discussions/1190#discussioncomment-11230229

**Comment by williambdean** [comment]
Custom prior are done with the `Prior` class. See the *latest* version of the docs: https://www.pymc-marketing.io/en/latest/notebooks/mmm/mmm_example.html

**Comment by williambdean** [comment, marked as answer]
"beta_channel" is stale and is not used anymore. Use "saturation_" prefixed variables for the mutliplying effect. That is usually "saturation_beta" but will be "saturation_alpha" for the Menten saturation. I have raised an issue here https://github.com/pymc-labs/pymc-marketing/issues/1256 to try to catch these earlier
